ush/create_aqm_rc_file.py

```diff
@@ -124,36 +123,11 @@ def create_aqm_rc_file(cdate, run_dir, init_concentrations):
     #
     #-----------------------------------------------------------------------
     #
-    with tempfile.NamedTemporaryFile(
-            dir="./",
-            mode="w+t",
-            prefix="aqm_rc_settings",
-            suffix=".yaml") as tmpfile:
-        tmpfile.write(settings_str)
-        tmpfile.seek(0)
-        cmd = " ".join(["uw template render",
-                "-i",
-                AQM_RC_TMPL_FP,
-                "-o",
-                aqm_rc_fp,
-                "-v",
-                "--values-file",
-                tmpfile.name,
-            ]
-        )
-        indent = "  "
-        output = ""
-        try:
-            output = check_output(cmd, encoding="utf=8", shell=True,
-                    stderr=STDOUT, text=True)
-        except CalledProcessError as e:
-            output = e.output
-            print(f"Failed with status: {e.returncode}")
-            sys.exit(1)
-        finally:
-            print("Output:")
-            for line in output.split("\n"):
-                print(f"{indent * 2}{line}")
+    render(
+        input_file = Path(AQM_RC_TMPL_FP),
+        output_file = Path(aqm_rc_fp),
+        values_src = settings,
+    )
```

Comment:
These same simplifications are repeated throughout these 5 files. Further info on `template.render()` can be found on the uwtools [Read the Docs](https://uwtools.readthedocs.io/en/2.1.1/sections/user_guide/api/template.html).